flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/StreamExecutionEnvironment.scala

```diff
@@ -615,7 +615,7 @@ class StreamExecutionEnvironment(javaEnv: JavaEnv) {
     
     val cleanFun = scalaClean(function)
     val typeInfo = implicitly[TypeInformation[T]]
-    asScalaStream(javaEnv.addSource(cleanFun).returns(typeInfo))
+    asScalaStream(javaEnv.addSource(cleanFun, typeInfo))
```

Comment:
Is there a reason why the `returns` call was removed here? This has broken our microservices we have upgraded to 1.9.1 since the output type of the transformer doesn't get set and later fails when the resulting DataStream is used.

Reply:
This change should actually still set the output type because we now pass the type directly in the function call which also sets it on the transformer? Could you maybe step through the code using a debugger and see if (or why not) the output type is set or not set.